npm/ux/index.ts

```diff
@@ -5,7 +5,25 @@
 // to a CSS file adjacent to the JS bundle and with the same name.
 import "./qsharp-ux.css";
 
+export { ColorMap } from "./colormap.js";
+export {
+  ColumnNames,
+  CreateReData,
+  InitialColumns,
+  ReDataToRow,
+  ReDataToRowScatter,
+  xAxis,
```

Comment:
Do all of these need to be exported at the top level of the package? Seems like some of these should be internal to the components. (e.g. does a consumer need to worry about xAxis and yAxis?)